llm4pddl/envs/assets/pddl/dressed/generate_tasks.py

```diff
@@ -0,0 +1,199 @@
+    for _ in range(num_extra_objects):
+        exs[rng.choice(extras)] += 1
+    # making objects:
+    persons = [f'person{i}' for i in range(1, num_people + 1 + exs['ex_p'])]
+    dresses = [
+        f'dress{i}' for i in range(1, num_formal_in_dress + 1 + exs['ex_d'])
+    ]
+    sweatpants = [
+        f'sweatpants{i}'
+        for i in range(1, num_casual_events + 1 + exs['ex_sp'])
+    ]
+    sweatshirts = [
+        f'sweatshirt{i}'
+        for i in range(1, num_casual_events + 1 + exs['ex_ss'])
+    ]
+    nicepants = [
+        f'nice-pants{i}'
+        for i in range(1, num_formal_in_suit + 1 + exs['ex_np'])
+    ]
+    collaredshirt = [
+        f'collared-shirt{i}'
+        for i in range(1, num_formal_in_suit + 1 + exs['ex_cs'])
+    ]
+    suitjacket = [
+        f'suit-jacket{i}'
+        for i in range(1, num_formal_in_suit + 1 + exs['ex_d'])
+    ]
+    objects = [{
+        'type': 'person',
+        'objs': persons
+    }, {
+        'type': 'dress',
+        'objs': dresses
+    }, {
+        'type': 'sweatpants',
+        'objs': sweatpants
+    }, {
+        'type': 'sweatshirt',
+        'objs': sweatshirts
+    }, {
+        'type': 'nice-pants',
+        'objs': nicepants
+    }, {
+        'type': 'collared-shirt',
+        'objs': collaredshirt
+    }, {
+        'type': 'suit-jacket',
+        'objs': suitjacket
+    }]
+    object_str = ''
+    for obj in objects:
+        if len(obj['objs']) > 0:
+            object_str += ' '.join(
+                obj['objs']) + f' - {obj["type"]}\n            '
+    # making initial predicates:
+    init_preds = ''
+    for person in persons:
+        init_preds += (f'(wearing-nothing-formal {person})\n         ' +
+                       f'(wearing-nothing-casual {person})\n         ')
+    for clothing in (dresses + sweatpants + sweatshirts + nicepants +
+                     collaredshirt + suitjacket):
+        init_preds += f'(in-closet {clothing})' + '\n         '
+    # making goal predicates:
+    goals = ''
+    goal_predicates = []
+    rng.shuffle(persons)
+    for i in range(num_casual_events):
+        goal_predicates.append(
+            f'(attending-casual-event {persons[i]})\n              ')
+    for i in range(
+            num_casual_events,
+        (num_casual_events + num_formal_in_dress + num_formal_in_suit)):
+        goal_predicates.append(
+            f'(attending-formal-event {persons[i]})\n              ')
+    rng.shuffle(goal_predicates)
+    for goal in goal_predicates:
+        goals += goal
+    #making problem:
+    problem_string = f"""(define (problem dressed)
+  (:domain dressed)
+  (:objects {object_str})
+  (:init {init_preds})
+  (:goal (and {goals}))
+  )"""
+    return problem_string
+
+
+def test_generate_dressing_problems() -> None:
+    """tests the function _generate_dressing_problems."""
+    rng = np.random.default_rng(seed=0)
+    prob1 = _generate_dressing_problems(2, 3, 2, 3, 0, 1, 0, 1, 0, 1, 1,
```

Comment:
can you add keyword arguments here and in all of the other invocations of this function to make it clear what the numbers correspond to? for example `_generate_dressing_problems(num_people=2, num_casual_events=3, ...)`. (you don't have to change anything about the function definition to do this.)